sql/mob_skill_lists.sql

```diff
@@ -774,7 +774,11 @@ INSERT INTO `mob_skill_lists` VALUES ('Moblin',184,1085);
 INSERT INTO `mob_skill_lists` VALUES ('Moblin',184,1086);
 INSERT INTO `mob_skill_lists` VALUES ('Moblin',184,1087);
 INSERT INTO `mob_skill_lists` VALUES ('Moblin',184,1088);
--- 185: Moogle
+INSERT INTO `mob_skill_lists` VALUES ('Jaded_Jody',185,316);  -- Impale
+INSERT INTO `mob_skill_lists` VALUES ('Jaded_Jody',185,317);  -- Vampiric Lash
+INSERT INTO `mob_skill_lists` VALUES ('Jaded_Jody',185,1793); -- Vampiric Root
+INSERT INTO `mob_skill_lists` VALUES ('Jaded_Jody',185,319);  -- Bad Breath
+INSERT INTO `mob_skill_lists` VALUES ('Jaded_Jody',185,320);  -- Sweet Breath
```

Comment:
Move this to the ASB section at the bottom of the skill list sql and keep 185: Moogle to be the same as LSB or add to the module.